src/Akka.Streams.Kafka/Akka.Streams.Kafka.csproj

```diff
@@ -4,7 +4,6 @@
     <AssemblyTitle>Akka.Streams.Kafka</AssemblyTitle>
     <Description>Apache Kafka adapter for Akka.NET Streams</Description>
     <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
-    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
This prevented the version of Akka.Streams.Kafka from being set by `RELEASE_NOTES.md`